specification/dartLangSpec.tex

```diff
@@ -13620,7 +13620,7 @@ \subsection{Assignable Expressions}
 
 <assignableSelector> ::= <unconditionalAssignableSelector>
   \alt `?.' <identifier>
-  \alt `?.[' <expression> `]'
+  \alt `?[' <expression> `]'
```

Comment:
Grammar should be 
```latex
\alt `?' `[' <expression> `]'
```
with `?` and `[` as separate tokens.

Reply:
Done